docs/fileset/README.md

```diff
@@ -212,15 +212,15 @@ Please, keep in mind when deleting metadata that we are not deleting the file it
 :::
 
 ## Setup Fileset for Digital Ocean
-Setup Digital Oceant space is very easy. As a first step, you need to create **public space** in Digital Ocean. How to do this you can find in this [tutorial](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key). After you need to do simple 3 steps: 
+The process of setting up a Digital Ocean Space is very easy. As a first step, you need to create a **public space**. You can see the full tutorial for this on [Digital Ocean](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key). After this, you need to do 3 simple steps: 
```

Comment:
Is a **Digital Ocean Space** a product provided by Digital Ocean? If so, it should be capitalised as it's a name. :smiley:

Reply:
I think you have to write it capitalized. **Spaces on DigitalOcean** is standing on their website

Reply:
@jeroenrinzema How would we go about talking about it in the sense of " a public space", would we change that to capitalized as well as I'm not sure whether it's 100% needed?

Reply:
public spaces can be with non-capital but we should respect naming from other companies. So if we use product name from another provider we should keep writing like it is on the official website.

Reply:
Okay, so are there any changes that need to be made to this before it is merged?